modules/ROOT/pages/hardware-and-software-requirements.adoc

```diff
@@ -1,40 +1,51 @@
-= Hardware and Software Requirements
+= Mule Runtime Engine Requirements for Hardware and Software
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :keywords: mule, requirements, jdk, installation
 
-Recommended minimum hardware for running Mule Runtime:
+== Minimum Hardware Requirements
+
+Mule Runtime Engine (Mule) requires minimum hardware resources:
 
 * 2GHz CPU or 1 virtual CPU in virtualized environments
 * 1GB of RAM
 * 4GB of storage
 
 Adjust RAM to match your latency requirements and the size and number of simultaneous messages that applications process.
 
-Software requirements are:
+== Minimum Software Requirements
```

Comment:
I broke this out into lists to make it easier to read, and to reinforce the idea that these are min. (supported) requirements, not quite the same as the table of tested things in the RN, per JIRA.